bundles/admin/admin-announcements/publisher/AnnouncementsTool.js

```diff
@@ -157,13 +158,13 @@ Oskari.clazz.define('Oskari.admin.bundle.admin-announcements.publisher.Announcem
 
             me.announcements.forEach(announcement => {
                 const announcementInput = me.templates.inputCheckbox.clone();
-                const annName = announcement.title;
+                const annName = announcement.locale[this.lang].name;
                 const annTime = announcement.begin_date.replace(/-/g, '/') + ' - ' + announcement.end_date.replace(/-/g, '/');
 
                 const idPrefix = 'oskari_announcement_select_';
                 announcementInput.find('input[type=checkbox]').attr({
                     'id': idPrefix + announcement.id,
-                    'value': announcement.title
+                    'value': announcement.locale[this.lang].name
```

Comment:
Isn't this the same value that is put on annName variable a few lines above?
```suggestion
                    'value': annName
```